src/common/main/java/me/owdding/skyocean/mixins/features/CloudRendererMixin.java

```diff
@@ -0,0 +1,20 @@
+package me.owdding.skyocean.mixins.features;
+
+import me.owdding.skyocean.config.features.misc.MiscConfig;
+import net.minecraft.client.renderer.CloudRenderer;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(CloudRenderer.class)
```

Comment:
should inject into LevelRenderer#renderLevel and wrap the check operation for if clouds are disabled instead, as fabric has its own clould renderer that mods can use and it doesnt call the net.minecraft.client.renderer.CloudRenderer for those.